chains/evm/contracts/libraries/RateLimiter.sol

```diff
@@ -116,14 +109,13 @@ library RateLimiter {
   }
 
   /// @notice Validates the token bucket config.
-  function _validateTokenBucketConfig(Config memory config, bool mustBeDisabled) internal pure {
+  function _validateTokenBucketConfig(
+    Config memory config
+  ) internal pure {
     if (config.isEnabled) {
-      if (config.rate >= config.capacity || config.rate == 0) {
```

Comment:
Why would setting the rate to zero pause the lane? It should just make the bucket unable to refill but if the capacity is already >0 it shouldn't have any affect?

Reply:
Capacity and rate can now be 0, which stops the lane